src/vcf/assembly_checker.cpp

```diff
@@ -23,6 +23,19 @@ namespace ebi
     namespace assembly_checker
     {
 
+      Record_Core build_record_core(std::string const & line)
```

Comment:
can we use constructor overloading instead ? one for (line) and another for (line, chromosome, position, reference_allele, alternate_allele)

Reply:
It is better not to make objects aware of the source they are constructed from, but receive the appropriate parameters.